src/main/resources/templates/markdown/nextSteps.md

```diff
@@ -26,23 +26,28 @@ Payment summary
 
 **Payment reference number** {{solsPaymentReferenceNumber}}
 
-What you need to do now:
+Notes for this application
+-------------------------------------------------
+
+{{additionalInfo}}
+
+What you need to do now
 ==================================================
 
-You will now need to send us:
+Print a copy of this page for your records. 
```

Comment:
we are just awaiting review from Janet regarding additional information on HOW to print this page, will discuss with her tomorrow and update this PR

Reply:
Confirmed from Janet, no additional information is required